style.css

```diff
@@ -229,16 +237,6 @@ ul {
   text-align: center;
 }
 
-.card-img {
-  display: flex;
-  border-top-left-radius: 8px;
-  border-top-right-radius: 8px;
-  flex: 1.1;
-  background: #ebf0ee;
-  height: 199px;
-  width: 21px;
-}
-
 .main-works {
   margin-top: 2.89rem;
   font-family: 'Inter', sans-serif;
```

Comment:
- [ ] KIndly apply the font-family to the language list as the previous reviewer as suggested.

- Your design
<img width="464" alt="image" src="https://github.com/AlyAtalla/AlyAtalla.github.io/assets/41163571/0b78c8d7-b844-4b25-8544-12c59737a24c">

- Figma design
<img width="368" alt="image" src="https://github.com/AlyAtalla/AlyAtalla.github.io/assets/41163571/e486fe1a-bcd2-4e63-8025-4225f6e4f891">

- Also fix the missing icon in front of `See live`